contact-us Page

KavishaMistry opened the pull request:
## Describe your changes
I created the contact us page and it is responsive for mobile and desktop designs. I have also created a text area component  
## Upload a screen recording of feature/bugfix in action (only requred if you did anything related to UI)

## Checklist before requesting a review
- [yes ] I have tested my code, everything is working fine and I did not break anything.
- [ yes] I have uploaded a screen recording of feature/bugfix if required.

https://github.com/NextPr0gram/UrbanThreads-Team-project-7/assets/148207321/b48c8ee5-9b6a-46d2-9abd-30626d70e552

NextPr0gram reviewed (changes requested):
![image](https://github.com/NextPr0gram/UrbanThreads-Team-project-7/assets/80346666/88a0a4a7-221e-4636-a12e-e6e291bd16cc)
Fix the spacing on the left of the heading, , don't use x slot as i said yesterday in the discord, just use "<h2>"

![image](https://github.com/NextPr0gram/UrbanThreads-Team-project-7/assets/80346666/a38ce8ef-2f39-4774-88d0-6391d3466b03)

Spacing is wrong here and the title should be outside the box, like in figma

![image](https://github.com/NextPr0gram/UrbanThreads-Team-project-7/assets/80346666/5d094569-1932-4ec9-a365-2250f548f7be)

Fix the spacing here it should be 8px like in figma.

There might be other spacing and sizing issues i cannot see.
Please make it exactly 100% like the one in figma. thx.

After the changes drop a comment in this pull request, no need to make a new one.

KavishaMistry commented:
I have altered the heading and fixed the spacing where necessary 

https://github.com/NextPr0gram/UrbanThreads-Team-project-7/assets/148207321/4ce7709a-e3d7-4676-af6f-f3836726af3d

KishanJ22 commented on resources/views/contact-us.blade.php:
```diff
@@ -0,0 +1,119 @@
+
+        <h2 class="text-lg font-formula1 py-3">Contact Us</h2>
+
+    <div class="grid grid-cols-1 md:grid-cols-2 grid-rows-1 gap-4 ">
```
Add padding to the grid. See how you can add padding here: [Tailwind CSS: Padding](https://tailwindcss.com/docs/padding).

KishanJ22 reviewed (changes requested):
Make sure the page looks exactly like the Figma design and look through the [Tailwind CSS documentation](https://tailwindcss.com/docs/installation) to fix the spacing and sizing issues that are present on the page.

Also, could you send a screenshot of what the whole page looks like please, because I can't see what the form looks like?

NextPr0gram commented:
Make sure the top "contact us heading" is the correct size.

![image](https://github.com/NextPr0gram/UrbanThreads-Team-project-7/assets/80346666/c55c5524-1446-4268-bf37-78815b186b42)
This text section should not overflow to the right column

KavishaMistry commented:
I fixed the heading of contact us to make it bigger, fixed the padding in various places and fixed the overflow of text

https://github.com/NextPr0gram/UrbanThreads-Team-project-7/assets/148207321/c8d26a3a-bccf-4052-8b81-98d8b848e607
